Sources/Tagged/Tagged.swift

```diff
@@ -56,7 +58,21 @@ extension Tagged: Equatable where RawValue: Equatable {
   }
 }
 
-extension Tagged: Error where RawValue: Error {
+extension Tagged: Error where RawValue: Error {}
```

Comment:
if we import Foundation we can override localizedDescription here, but it doesnt work (never gets called)